lib/matplotlib/patches.py

```diff
@@ -791,22 +791,25 @@ def set_xy(self, xy):
         self.stale = True
 
     def set_width(self, w):
-        "Set the width of the rectangle."
+        """Set the width of the rectangle."""
         self._width = w
         self._update_x1()
         self.stale = True
 
     def set_height(self, h):
-        "Set the height of the rectangle."
+        """Set the height of the rectangle."""
         self._height = h
         self._update_y1()
         self.stale = True
 
     def set_bounds(self, *args):
         """
-        Set the bounds of the rectangle.
+        Set the bounds of the rectangle as *left*, *bottom*, *width*, *height*.
+
+        The values may be passed as separte parameters or as a tuple::
```

Comment:
Not trying to define a `Parameters` section here, which would be quite awkward given that this only has positional arguments. and they may be one or four.

Nevertheless, the new docstring is an improvement IMHO.